Tests/SwiftGenKitTests/CatalogEntryTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CatalogEntryTests.swift
+//  Tests
+//
+//  Created by Francisco Diaz on 10/5/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftGenKit
+
+final class CatalogEntryTests: XCTestCase {
+  func testGroupEntry_IsComparableAlphabetically() {
+    let entryA = Catalog.Entry.group(name: "A", items: [])
+    let entryB = Catalog.Entry.group(name: "B", items: [])
+
+    XCTAssertEqual(entryA < entryB, true)
```

Comment:
The expected usage of `XCTAssertEqual` is to take 2 parameters to equate, like `XCTAssertEqual(entryA, entryB)`.

If you want to check if an expression, like `entryA < entryB`, is true, prefer `XCTAssert` or `XCTAssertTrue`.
_(Some even prefer using `XCTAssertLessThan(entryA, entryB)` but this last one being less commonly known, `XCTAssert` or `XCTAssertTrue` are generally more fitting)_